toolflow/vivado/platform/pcie/pcie_base.tcl

```diff
@@ -17,358 +17,365 @@
+
+  # create hierarchical ports: resets
+  set ddr_aresetn [create_bd_pin -type "rst" -dir "O" "ddr_aresetn"]
+  set design_aresetn [create_bd_pin -type "rst" -dir "O" "design_aresetn"]
+  set pcie_p_aresetn [tapasco::subsystem::get_port "host" "rst" "peripheral" "resetn"]
+  set ddr_ic_aresetn [tapasco::subsystem::get_port "mem" "rst" "interconnect"]
+  set ddr_p_aresetn  [tapasco::subsystem::get_port "mem" "rst" "peripheral" "resetn"]
+  set design_p_aresetn [tapasco::subsystem::get_port "design" "rst" "peripheral" "resetn"]
+
+  # create instances of cores: MIG core, DMA
+  set mig [create_mig_core "mig"]
+
+  variable pcie_width
+  if { $pcie_width == "x8" } {
+    set dma [tapasco::ip::create_bluedma "dma"]
+  } else {
+    set dma [tapasco::ip::create_bluedma_x16 "dma"]
+  }
+  connect_bd_net [get_bd_pins $dma/IRQ_read] [::tapasco::ip::add_interrupt "PLATFORM_COMPONENT_DMA0_READ" "host"]
+  connect_bd_net [get_bd_pins $dma/IRQ_write] [::tapasco::ip::add_interrupt "PLATFORM_COMPONENT_DMA0_WRITE" "host"]
+
+  set mig_ic [tapasco::ip::create_axi_sc "mig_ic" 2 1]
+  tapasco::ip::connect_sc_default_clocks $mig_ic "mem"
+
+  connect_bd_intf_net [get_bd_intf_pins mig_ic/M00_AXI] [get_bd_intf_pins -regexp mig/(C0_DDR4_)?S_AXI]
+
+  # AXI connections:
+  # connect dma 32bit to mig_ic
+  connect_bd_intf_net [get_bd_intf_pins $dma/M32_AXI] [get_bd_intf_pins mig_ic/S00_AXI]
+  # connect DMA 64bit to external port
+  connect_bd_intf_net [get_bd_intf_pins $dma/M64_AXI] $m_axi_mem
+  # connect second mig_ic slave to external port
+  connect_bd_intf_net $s_axi_mem [get_bd_intf_pins mig_ic/S01_AXI]
+  # connect DMA S_AXI to external port
+  connect_bd_intf_net $s_axi_ddma [get_bd_intf_pins $dma/S_AXI]
+
+  # connect PCIe clock and reset
+  connect_bd_net $pcie_aclk [get_bd_pins $dma/m64_axi_aclk] [get_bd_pins $dma/s_axi_aclk]
+  connect_bd_net $pcie_p_aresetn [get_bd_pins $dma/m64_axi_aresetn] [get_bd_pins $dma/s_axi_aresetn]
+
+  # connect DDR clock and reset
+  set ddr_clk [get_bd_pins -regexp mig/(c0_ddr4_)?ui_clk]
+  connect_bd_net [tapasco::subsystem::get_port "mem" "clk"] \
+    [get_bd_pins $dma/m32_axi_aclk]
+  connect_bd_net $ddr_p_aresetn \
+    [get_bd_pins $dma/m32_axi_aresetn] \
+    [get_bd_pins -regexp mig/(c0_ddr4_)?aresetn]
+
+  # connect external DDR clk/rst output ports
+  connect_bd_net $ddr_clk $ddr_aclk
+
+  set design_clk_wiz [tapasco::ip::create_clk_wiz design_clk_wiz]
+  set_property -dict [list CONFIG.CLK_OUT1_PORT {design_clk} \
+    CONFIG.USE_SAFE_CLOCK_STARTUP {false} \
+    CONFIG.CLKOUT1_REQUESTED_OUT_FREQ [tapasco::get_design_frequency] \
+    CONFIG.USE_LOCKED {true} \
+    CONFIG.USE_RESET {true} \
+    CONFIG.RESET_TYPE {ACTIVE_LOW} \
+    CONFIG.RESET_PORT {resetn} \
+    CONFIG.PRIM_SOURCE {No_buffer} \
+    ] $design_clk_wiz
+
+  connect_bd_net [get_bd_pins $design_clk_wiz/resetn] [get_bd_pins -regexp $mig/((init_calib_complete)|(c0_init_calib_complete))]
```

Comment:
Why did you change this line? (also included `mmcm_locked` as possible ouput signal for the MIG before)
This breaks the new U50 because there the "MIG" has no `init_calib_complete` only the `mmcm_locked` output

Reply:
I have to check what happened here. My U50.tcl creates an `mmcm_locked` output and I've been using it for quite some time successfully... Might have been some rebase issue with the master branch.